### DISCUSSION TITLE
求助，Qwen2.5的函数调用输出结构是什么样子的

### DISCUSSION THREAD

**Discussion by akai-shuuichi** [QwenLM/Qwen3#1073] [Q&A] [answered]
目前遇到这几种情况
1. 文本<tools_call>tool信息</tools_call>
2. <tools_call>tool信息</tools_call>文本
3.文本<tools_call>tool信息</tools_call><tools_call>tool信息</tools_call>
请问还有什么格式吗，tools之间可能会有文本吗

**Comment by jklj077** [comment, marked as answer]
正常情况下只有
```
<tool_call>
JSON-formatted string
</tool_call>
```
看这里https://qwen.readthedocs.io/en/latest/framework/function_call.html#qwen2-5-function-calling-templates

其它情况不符合预期，可以扔badcase到issue。
